<ui version="4.0" >
 <comment>*********************************************************************
**
** Copyright (C) 1992-$THISYEAR$ $TROLLTECH$. All rights reserved.
**
** This file is part of the $MODULE$ of the Qt Toolkit.
**
** $TROLLTECH_DUAL_LICENSE$
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <class>QtGradientStopsEditor</class>
 <widget class="QWidget" name="QtGradientStopsEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="5" column="1" >
    <widget class="QSpinBox" name="valueSpinBox" >
     <property name="maximum" >
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4" >
    <widget class="qdesigner_internal::QtSpinBox" name="zoomSpinBox" >
     <property name="suffix" >
      <string>%</string>
     </property>
     <property name="maximum" >
      <number>10000</number>
     </property>
     <property name="minimum" >
      <number>100</number>
     </property>
     <property name="singleStep" >
      <number>100</number>
     </property>
     <property name="value" >
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="qdesigner_internal::QtDoubleSpinBox" name="positionSpinBox" >
     <property name="decimals" >
      <number>3</number>
     </property>
     <property name="maximum" >
      <double>1</double>
     </property>
     <property name="minimum" >
      <double>0</double>
     </property>
     <property name="singleStep" >
      <double>0.01</double>
     </property>
     <property name="value" >
      <double>0</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QSpinBox" name="hueSpinBox" >
     <property name="maximum" >
      <number>359</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QSpinBox" name="alphaSpinBox" >
     <property name="maximum" >
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QSpinBox" name="saturationSpinBox" >
     <property name="maximum" >
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="6" >
    <widget class="QFrame" name="frame_4" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="qdesigner_internal::QtColorLine" name="alphaColorLine" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>13</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="2" colspan="6" >
    <widget class="QFrame" name="frame_3" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="qdesigner_internal::QtColorLine" name="valueColorLine" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>13</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2" colspan="6" >
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="qdesigner_internal::QtColorLine" name="saturationColorLine" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>13</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="8" >
    <widget class="qdesigner_internal::QtGradientStopsWidget" name="gradientWidget" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>13</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="positionLabel" >
     <property name="text" >
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" >
    <widget class="QLabel" name="zoomLabel" >
     <property name="text" >
      <string>Zoom</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="colorLabel" >
     <property name="text" >
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7" >
    <widget class="QToolButton" name="zoomAllButton" >
     <property name="text" >
      <string>Zoom All</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="hueLabel" >
     <property name="text" >
      <string>Hue</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="saturationLabel" >
     <property name="text" >
      <string>Saturation</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="valueLabel" >
     <property name="text" >
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="alphaLabel" >
     <property name="text" >
      <string>Alpha</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6" >
    <widget class="QToolButton" name="zoomOutButton" >
     <property name="text" >
      <string>Zoom Out</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5" >
    <widget class="QToolButton" name="zoomInButton" >
     <property name="text" >
      <string>Zoom In</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="7" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="qdesigner_internal::QtColorButton" name="colorButton" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="hsvRadioButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>HSV</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rgbRadioButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>RGB</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2" colspan="6" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="qdesigner_internal::QtColorLine" name="hueColorLine" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>13</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qdesigner_internal::QtDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qtspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>qdesigner_internal::QtGradientStopsWidget</class>
   <extends>QFrame</extends>
   <header>qtgradientstopswidget.h</header>
  </customwidget>
  <customwidget>
   <class>qdesigner_internal::QtSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qtspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>qdesigner_internal::QtColorLine</class>
   <extends>QFrame</extends>
   <header>qtcolorline.h</header>
  </customwidget>
  <customwidget>
   <class>qdesigner_internal::QtColorButton</class>
   <extends>QToolButton</extends>
   <header>qtcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>gradientWidget</tabstop>
  <tabstop>positionSpinBox</tabstop>
  <tabstop>zoomSpinBox</tabstop>
  <tabstop>zoomInButton</tabstop>
  <tabstop>zoomOutButton</tabstop>
  <tabstop>zoomAllButton</tabstop>
  <tabstop>colorButton</tabstop>
  <tabstop>hsvRadioButton</tabstop>
  <tabstop>rgbRadioButton</tabstop>
  <tabstop>hueColorLine</tabstop>
  <tabstop>saturationColorLine</tabstop>
  <tabstop>valueColorLine</tabstop>
  <tabstop>alphaColorLine</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
